plugins.rb

```diff
@@ -1,3 +1,5 @@
 # frozen_string_literal: true
 
-require_relative "lib/bootboot"
+$LOAD_PATH.unshift(File.expand_path("../lib", __FILE__))
```

Comment:
Do we need to change the LOAD_PATH?

Reply:
Good point, no we don't need to change `LOAD_PATH`, will fix.